web/contact/base_test.go

```diff
@@ -30,6 +30,20 @@ func TestCreate(t *testing.T) {
 	testsuite.RunWebTests(t, ctx, rt, "testdata/create.json", nil)
 }
 
+func TestDeindex(t *testing.T) {
+	ctx, rt := testsuite.Runtime()
+
+	defer func() {
+		rt.DB.MustExec(`UPDATE contacts_contact SET is_active = true WHERE id IN ($1, $2)`, testdata.Bob.ID, testdata.George.ID)
+	}()
+
+	rt.DB.MustExec(`UPDATE contacts_contact SET is_active = false WHERE id IN ($1, $2)`, testdata.Bob.ID, testdata.George.ID)
+
+	defer testsuite.Reset(testsuite.ResetElastic)
```

Comment:
This can be moved to the anon function defer above?

Reply:
I think this is why the test was failing.. defers are executed in LIFO order so reindex was running before the contacts were marked as active.